pkg/server/handler.go

```diff
@@ -184,19 +78,6 @@ func WithAuditAnnotation(handler http.Handler) http.HandlerFunc {
 	})
 }
 
-// WithClusterAnnotation adds the cluster name into the annotation of an audit
-// event. Needs initialized annotations.
-func WithClusterAnnotation(handler http.Handler) http.HandlerFunc {
```

Comment:
this needs a better name. WithAuditEventClusterAnnotation

Reply:
updated.